R/inside.R

```diff
@@ -72,43 +72,39 @@
 #' @importFrom magrittr inset2
 #' 
 #' @export
-inside <- function(multiverse, .expr) {
-  .expr = enexpr(.expr)
+inside <- function(.m, .code) {
+  .code = enexpr(.code)
+  multiverse = attr(.m, "multiverse")
   
-  if (is_null(multiverse@code)) {
-    multiverse@code <- .expr
-  } else {
-    
-    multiverse@code <- multiverse@code %>% 
-      inset2(., length(.) + 1, .expr[[2]])
-  }
-  
-  multiverse
+  add_and_parse_code(multiverse, .code)  
 }
 
-#' @export
-`$<-.multiverse` <- function(multiverse, name, value) {
-  .expr = expr({ !!sym(name) <- !!rlang::f_rhs(value) })
+`$<-.multiverse` <- function(.m, name, value) {
```

Comment:
At some point (maybe not now --- in which case, merge this PR but create an issue so we don't forget) we should probably change the name of the classes we use so that they would avoid collisions with names used in other packages. Easiest way to do that would be to prefix with the package name, so the `multiverse` class would become the `multidy_multiverse` class. Something like that.